Mathlib/Order/Basic.lean

```diff
@@ -1156,6 +1156,21 @@ lemma eq_or_eq_or_eq_of_forall_not_lt_lt [LinearOrder α]
   rcases hne.2.2.lt_or_gt with h₃ | h₃
   exacts [h h₁ h₂, h h₂ h₃, h h₃ h₂, h h₃ h₁, h h₁ h₃, h h₂ h₃, h h₁ h₃, h h₂ h₁]
 
+namespace Empty
+
+def linearOrder (α : Type*) [inst : IsEmpty α] : LinearOrder α where
+  le _ _ := False
+  le_refl x := inst.false x
+  le_trans x _ _ := (inst.false x).elim
+  le_antisymm x := (inst.false x).elim
+  le_total x _ := (inst.false x).elim
+  toDecidableLE x _ := Decidable.isFalse (inst.false x).elim
+
+attribute [instance] linearOrder Empty
```

Comment:
This makes `linearOrder` and `Empty` into instances. Definitely not what you want!